SimpleAnnPlayground/Debugging/FrmElementDesigner.Designer.cs

```diff
@@ -104,8 +104,8 @@ private void InitializeComponent()
             // 
             // PicDraw
             // 
+            this.PicDraw.Anchor = System.Windows.Forms.AnchorStyles.None;
```

Comment:
This should be removed to use the `Dock` property.

Reply:
Same case, I'll fix it